README.md

```diff
@@ -14,7 +14,7 @@
 
 healthchecks is a watchdog for your cron jobs. It's a web server that listens for pings from your cron jobs, plus a web interface.
 
```

Comment:
Good cool idea noting this change, although going through the whole file there are a lot of things that can be changed.

Reply:
Great work.The title of PR,I think it will be better if you include the pivotal tracker ID also